packages/beacon-node/src/network/peers/discover.ts

```diff
@@ -406,7 +406,15 @@ export class PeerDiscovery {
     }
   }
 
+  private isPeerCoolingDown(peer: CachedENR): boolean {
+    const peerScore = this.peerRpcScores.getScore(peer.peerId);
+    return peerScore <= -1;
```

Comment:
maybe just check `< 0`, it's just the same. Otherwise need to use `SCORE_THRESHOLD` to give more context about this value

also add comment saying we assume to call this method for a disconnected peer, ie on PeerDiscovery
(it makes no sense to decide a cooling down peer just based on a score)

Reply:
Ok, just double checking my logic here.  I left it -1 so it would account for decaying scores before reseting them.  Looking at the logic in `PeerRpcScoreStore.update` again I see that we update all individual scores and then prune ones below `Math.abs(score) < 1` synchronously.  So for peers below `1` they delete on that round of updates after decay less than `-1`.  On the next `getScore` the peer will be missing so the default value of 0 gets returned.  Checking for `0` is a safe assumption here.